propolis/src/hw/qemu/ramfb.rs

```diff
@@ -151,12 +151,32 @@ impl Migrate for RamFb {
             stride: state.stride,
         })
     }
+
+    fn import(
+        &self,
+        _dev: &str,
+        deserializer: &mut dyn erased_serde::Deserializer,
+        _ctx: &DispCtx,
+    ) -> Result<(), crate::migrate::MigrateStateError> {
+        let deserialized: migrate::RamFbV1 =
+            erased_serde::deserialize(deserializer)?;
```

Comment:
Not an issue with this PR itself, but more of a general question: Do we have plans for a design of how device logic is going to discriminate between different versions of its migration payload on import?

Reply:
I've been slapping `V1` on struct names as I do 'em, assuming we'll have some mechanism for that discrimination, but I'm not as familiar with the bits actually driving the data transfer to know how the downstream logic will make that decision prior to attempting a deserialization

Reply:
There isn't any concrete plans AFAIK since we've been focused mostly on just getting it working initially. I was thinking we'd have an approach similar to [this](https://outcrawl.com/event-versioning-rust). Something like that plus refactoring to easily generate the structures for every device.

Reply:
I guess there's a question about how local those kinda version decision get made: are we allowing sending device state with independent versioning or versioning an entire set?

Reply:
I think the guidance for V1 is to throw a version number on everything. I'm not convinced they're all necessary (I have convinced myself that the easiest path forward is to have a central repository of types keyed by a global version number), but it'll be easier to rev them out over time than to try and rev them in.

Reply:
@luqmana that version tagging looks like it could be a good start.

There's no harm in slapping V1 on structs for now.  If we decide on a different path, it's not like they'll be harmful to clean up at that point.